src/renderer/views/Playlist/Playlist.js

```diff
@@ -340,7 +340,7 @@ export default defineComponent({
         this.playlistDescription = result.info.description ?? ''
         this.firstVideoId = result.items[0].id
         this.playlistThumbnail = result.info.thumbnails[0].url
-        this.viewCount = extractNumberFromString(result.info.views)
+        this.viewCount = result.info.views.toLowerCase() === 'no views' ? 0 : extractNumberFromString(result.info.views)
```

Comment:
**suggestion (non-blocking):** We have a few places in the code where we use `extractNumberFromString` and it ends up getting used as `NaN`. I get from an "API" level why `NaN` is an appropriate return value, but it is this behavior that ultimately results in us displaying `NaN` throughout the app. I'd recommend returning `0` instead of `NaN` from that function in both the not-string case and `NaN` `parseInt` case just to cover the swathe of edge cases more easily.

Reply:
While you are correct that in many places we don't handle `NaN` and just display it to the user, in other places we actually rely on it, so we know when to hide UI elements for example.

So we either need to create a separate wrapper function the converts the `NaN` to zero in the places where that is desired or add an extra parameter to control when the fallback should be zero.